pages/api/join.js

```diff
@@ -57,18 +57,8 @@ export default async function handler(req, res) {
     }
   }
 
-  await joinTable.create({
```

Comment:
Let's keep this around for now– at the very least it lets us keep a waitlist & prevents us from breaking another system under the hood that relies on this but forgot. To speed this up, what if we run this in parallel (`Promise.all`) with the toriel endpoint?